Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -505,10 +506,10 @@ static u32 ComputeNameSize(const File::FSTEntry& parent_entry)
   return name_size;
 }
 
-static std::string ASCIIToLowercase(std::string str)
+static std::string ASCIIToUppercase(std::string str)
 {
   std::transform(str.begin(), str.end(), str.begin(),
-                 [](char c) { return std::tolower(c, std::locale::classic()); });
+                 [](char c) { return std::toupper(c, std::locale::classic()); });
```

Comment:
In the commit message for this change, you said that the old sorting order was invalid. Invalid based on what? Does Nintendo define a canonical sorting order for FSTs? I don't mind using uppercase instead of lowercase, but I'd like to understand the reason for the change.